inclusive-design/wecount.inclusivedesign.ca
# feat: update readability values

## Description

Changed readability values saved into physical files: B -> Beginner, I -> Intermediate, E -> Expert

This PR should be merged before https://github.com/BlueSlug/wecount.inclusivedesign.ca/pull/5 so the https://github.com/BlueSlug/wecount.inclusivedesign.ca/pull/5 can be tested with modified data files.

## Steps to test

1. Make sure all existing resources files have these values changed;
2. Go to the admin site, change "readability" of one or more resources;
3. Make sure saved readability values are in new values;
4. Create a new resource via the admin interface;
5. Make sure saved readability values are in new values.

**Expected behavior:** 
See above.
